pkg/R/sparkR.R

```diff
@@ -15,7 +15,7 @@ sparkR.onLoad <- function(libname, pkgname) {
 # backend and FALSE otherwise
 connExists <- function(env) {
   tryCatch({
-    exists(".sparkRCon", envir = env) && isOpen(env[[".sparkRCon"]])
+    !exists(".sparkRCon", envir = env) || (exists(".sparkRCon", envir = env) && isOpen(env[[".sparkRCon"]]))
```

Comment:
yeah, sparkR.stop() may be called multiple times. But this change looks confusing.

I suggest to delete the connExists() function, and replace the line:
  if (!connExists(env)) {
with 
    if (exists(".sparkRCon", envir = env) && !isOpen(env[[".sparkRCon"]]) {

@shivaram , what do you think?